### ISSUE TITLE
Mesen 0.9.8 Hangs/Freezes in Windows 10

### ISSUE THREAD

**Issue by Binarynova** [SourMesen/Mesen#726]
Version: Mesen 0.9.8
OS: Windows 10.

When I close the application the program hangs for several seconds (long enough for Windows to displays the (Not Responding) message in the title bar) before finally closing.

I'm not sure if it's related but I also see hanging/freezing behavior when trying to Reset the current ROM.

This is on a fresh install of Windows 10 done yesterday, with updates and video drivers applied, though I have also had this issue on a previous installation of Windows 10.

**Comment by SourMesen** [comment]
Thanks for the report.
Does this happen with any ROM?  Does it happen 100% of the time?
Could you try older versions of Mesen (e.g 0.9.7, 0.9.6) to see if they behave any differently?

**Comment by Binarynova** [comment]
I went back to confirm the issue in 0.9.8 before downloading other versions so I'd have a fresh memory to compare with. Turns out it's not happening now. It actually may have been that my Windows install was **too** fresh. I don't think I had installed DirectX, for example. Either way I'm no longer getting the freezing when I try to close the emulator, nor when I use menu options like Reset.

Sorry about that. :) Thanks for reaching out!
